src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -52,102 +28,28 @@ def post(self, request: Request) -> Response:
+            dummy_occurrence = dict(load_data("generic-event-profiling"))
+            dummy_occurrence["event"]["received"] = datetime.utcnow().isoformat()
+            dummy_occurrence["event"]["timestamp"] = datetime.utcnow().isoformat()
             user = user_service.get_user(request.user.id)
             projects = Project.objects.get_for_user_ids({user.id})
             if not projects:
                 return Response(
                     "Requesting user must belong to at least one project.",
                     status=status.HTTP_400_BAD_REQUEST,
                 )
-            event = {
-                "event_id": "44f1419e73884cd2b45c79918f4b6dc4",
-                "project_id": projects[0].id,
-                "platform": "python",
-                "tags": {"environment": "prod"},
-                "timestamp": ensure_aware(datetime.now()),
-                "received": ensure_aware(datetime.now()),
-                "data": {
-                    "stacktrace": {
-                        "frames": [
-                            {"function": "0x0", "in_app": False},
-                            {"function": "start_sim", "in_app": False},
-                            {"function": "main", "in_app": False, "package": "xctest"},
-                        ]
-                    },
-                },
-            }
-        else:
-            event = request.data.pop("event", None)
-
-        if not event:
-            return Response(
-                "Must pass an event or query param of dummyEvent=True",
-                status=status.HTTP_400_BAD_REQUEST,
-            )
-
-        if request.query_params.get("dummyOccurrence") == "True":
-            occurrence = {
-                "id": "55f1419e73884cd2b45c79918f4b6dc5",
-                "project_id": event["project_id"],
-                "fingerprint": ["some-fingerprint"],
-                "issue_title": "something bad happened",
-                "subtitle": "it was bad",
-                "resource_id": "1234",
-                "evidence_data": {"Test": 123},
-                "evidence_display": [
-                    {
-                        "name": "Attention",
-                        "value": "Very important information!!!",
-                        "important": True,
-                    },
-                    {
-                        "name": "Evidence 2",
-                        "value": "Not important",
-                        "important": False,
-                    },
-                    {
-                        "name": "Evidence 3",
-                        "value": "Nobody cares about this",
-                        "important": False,
-                    },
-                ],
-                "type": ProfileBlockedThreadGroupType.type_id,
-                "detection_time": ensure_aware(datetime.now()),
-                "event": event,
-            }
+            dummy_occurrence["project_id"] = projects[0].id
```

Comment:
This would force the creation of a profile issue on an arbitrary project the caller is a member of? I think that's fine since itll just be us calling it?

Reply:
Yeah, that's just the original logic. I've provided an override since it's not always ideal